lib/segment/src/data_types/vectors.rs

```diff
@@ -243,6 +264,16 @@ pub struct NamedVector {
     pub vector: VectorType,
 }
 
+/// Vector data with name
+#[derive(Debug, Deserialize, Serialize, JsonSchema, Clone)]
```

Comment:
Needs to be added to `schema_generator.rs`

Reply:
fixed